.github/workflows/master.yml

```diff
@@ -59,19 +58,19 @@ jobs:
         toolchain: ${{ matrix.toolchain }}
         args: --target ${{ matrix.target }} --workspace
 
-    # If musl, compile and test all excluding kafka
+    # If musl, compile and test all
     - uses: actions-rs/cargo@v1
       if: matrix.target == 'x86_64-unknown-linux-musl'
       with:
-        command: build
         toolchain: ${{ matrix.toolchain }}
-        args: --target ${{ matrix.target }} --workspace --exclude cloudevents-sdk-rdkafka
+    - run: CC=musl-gcc CXX=g++ cargo build --target ${{ matrix.target }} --all
+      if: matrix.target == 'x86_64-unknown-linux-gnu'
     - uses: actions-rs/cargo@v1
       if: matrix.target == 'x86_64-unknown-linux-musl'
       with:
-        command: test
         toolchain: ${{ matrix.toolchain }}
-        args: --target ${{ matrix.target }} --workspace --exclude cloudevents-sdk-rdkafka
+    - run: CC=musl-gcc CXX=g++ cargo test --target ${{ matrix.target }} --all
```

Comment:
Why? It's tested above

Reply:
that was a build run. If you want we could directly test

Reply:
this is a build run using glibc target, we don't need it it's already above